### ISSUE TITLE
upload without public directory 

### ISSUE THREAD

**Issue by shahriar1** [Vinelab/cdn#104]
I want to upload assets that exists in public directory but not the public directory itself. Is it possible by changing any config?

**Comment by Mulkave** [comment]
Yes, you can use a combination of `include` and `exclude` from the docs https://github.com/Vinelab/cdn#files--directories to choose specifically what you'd like to have uploaded. Also when configuring your buckets, you may specify that:

```php
'buckets' => [

    'my-bucket' => 'public/*'',

]
```

**Comment by shahriar1** [comment]
I tried changing those options but it does upload public directory itself too with the files under public directory. But I only want to upload inner contents of the public directory not the public directory itself.

**Comment by Mulkave** [comment]
Have you tried excluding the public directory? For it's the default in the package.

**Comment by shahriar1** [comment]
Yes I tried.  It seems by default it pushes public directory when I run push command

**Comment by benswinburne** [comment]
I'm also having this issue. Can anyone post a working include/exclude example to make this work please?
